.github/workflows/ci.yaml

```diff
@@ -23,3 +23,21 @@ jobs:
       - uses: arduino/setup-protoc@master
       - name: Build and test
         run: make
+      - uses: actions/upload-artifact@v2
+        with:
+          name: generated-pb-go-files
+          path: proto/go/degitxpb/
+  golangci:
```

Comment:
I've done it in separate jobs not because  of parallelism, I just can't do this to steps (proto generation & linters) together on one environment. I've tried it a dozen of times. If I call `golangci/golangci-lint-action@v2` in the same job after all actions that are needed to generate proto files, all linters perform well, but because of this error
```
.github#L1 
Tar failed with error: The process '/bin/tar' failed with exit code 2  
````
and thousands of
```
.github#L1
Cannot open: File exists
``` 
problem places wouldn't be annotated properly by golangci-lint and you need to deep dive in action report to understand why build failed. 
[corrupted report example](https://github.com/stepanvalyavskiy/degitx/actions/runs/302173663)
log snippet:
```
Installed golangci-lint into /home/runner/golangci-lint-1.31.0-linux-amd64/golangci-lint in 850ms
/bin/tar: ../../../go/pkg/mod/google.golang.org/grpc@v1.0.5/codegen.sh: Cannot open: File exists
Error: /bin/tar: ../../../go/pkg/mod/google.golang.org/grpc@v1.0.5/benchmark/client/main.go: Cannot open: File exists
Error: /bin/tar: ../../../go/pkg/mod/google.golang.org/grpc@v1.0.5/benchmark/benchmark.go: Cannot open: File exists
Error: /bin/tar: ../../../go/pkg/mod/google.golang.org/grpc@v1.0.5/benchmark/worker/util.go: Cannot open: File exists
Error: /bin/tar: ../../../go/pkg/mod/google.golang.org/grpc@v1.0.5/benchmark/worker/benchmark_server.go: Cannot open: File exists
Error: /bin/tar: ../../../go/pkg/mod/google.golang.org/grpc@v1.0.5/benchmark/worker/main.go: Cannot open: File exists
Error: /bin/tar: ../../../go/pkg/mod/google.golang.org/grpc@v1.0.5/benchmark/worker/benchmark_client.go: Cannot open: File exists
```